src/Terminal/HelpText.ts

```diff
@@ -148,8 +148,7 @@ export const HelpTexts: Record<string, string[]> = {
   cd: [
     "Usage: cd [dir]",
     " ",
-    "Change to the specified directory. Note that this works even for directories that don't exist. If you ",
-    "change to a directory that does not exist, it will not be 'created'. Examples:",
+    "Change to the specified directory. If you change to a directory that does not exist, it will not be created. Examples:",
```

Comment:
I'd say that trying to change to a directory that doesn't exist is an error. The current wording is weird, but made more sense back when you could cd to non-existing paths.